internal/ui/ui_js.go

```diff
@@ -129,6 +130,7 @@ func (u *userInterface) update(g GraphicsContext) error {
 	if err := g.Update(); err != nil {
 		return err
 	}
+	currentInput.runebuffer = currentInput.runebuffer[:0]
```

Comment:
? Is there a strong reason you want to use `currentInput.runebuffer[:0]` instead of `nil`?